src/node.cpp

```diff
@@ -283,4 +260,17 @@ Status Node::demultiplyOutputs(SessionResults& nodeSessionOutputs) {
     return StatusCode::OK;
 }
 
+Status Node::createShardedBlob(InferenceEngine::Blob::Ptr& dividedBlob, const InferenceEngine::TensorDesc& dividedBlobDesc, InferenceEngine::Blob::Ptr blob, size_t i, size_t step, const NodeSessionMetadata& metadata, const std::string blobName) {
+    auto status = createSharedBlob(dividedBlob, dividedBlobDesc);
```

Comment:
Here this is not clear that you use `createSharedBlob` from ov_utils. You can mistake it for class method